ruby/README.md

```diff
@@ -1,17 +1,27 @@
-# Ruby Demos
-
-## Overview
-
-This directory contains Ruby sample programs for various Ice components. These
-examples are provided to get you started on using a particular Ice feature or
-coding technique.
+# Ice for Ruby Demos
 
-Refer to the README.md file in each demo directory for usage instructions.
+This folder contains example Ice applications written in C#. Each example is a simple client-server application that
+demonstrates a specific feature or programming technique.
 
-## Prerequisites
+## Building Requirements
 
 You'll need to install Ice for Ruby in order to use any of the sample programs.
 
 ```shell
```

Comment:
In a follow-up PR we should create a `Gemfile` in each demo with appropriate instructions.